Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/util/GenericDBUtils.java

```diff
@@ -196,4 +200,37 @@ public static void rollbackTransaction(Transaction transaction) {
         }
         return queryList;
     }
+
+    public static <T> void debugApplicationContext(Class<T> from, ApplicationContext context) {
```

Comment:
Why do we have all of this in the GenericDBUtils? It has nothing to do with the database.